docs/resources/Auto_Moderation.md

```diff
@@ -0,0 +1,221 @@
+| SLURS            | 3     | Personal insults or words that may be considered hate speech |
+
+
+###### Event Types
+
+Indicates in what event context a rule should be checked.
+
+| Type             | Value   | Description                              |
+| ---------------- | ------- | ---------------------------------------- |
+| MESSAGE_SEND     | 1       | when a member sends a message on a guild |
+
+
+### Auto Moderation Action Object
+
+An action which will execute whenever a rule is triggered.
+
+###### Auto Moderation Action Structure
+
+| Field       | Type                                                                                             | Description                                                               |
+| ----------- | ------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------- |
+| type        | [action type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-types)        | the type of action                                                        |
+| metadata? * | [action metadata](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-metadata) | additional metadata needed during execution for this specific action type |
+
+\* Can be omitted based on `type`. See the `Associated Action Types` column in [action metadata](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-metadata) to understand which `type` values require `metadata` to be set.
+
+###### Action Types
+
+| Type                  | Value   | Description                                           |
+| --------------------- | ------- | ----------------------------------------------------- |
+| BLOCK_MESSAGE         | 1       | blocks the content of a message according to the rule |
+| SEND_ALERT_MESSAGE    | 2       | logs user content to a specified channel              |
+| TIMEOUT               | 3       | timeout user for a specified duration *               |
+
+\* `MODERATE_MEMBERS` permission is required to use this action type
+
+
+###### Action Metadata
+
+Additional data used when an action is executed. Different fields are relevant based on
+value of [action type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-types).
+
+| Field            | Type       | Associated Action Types | Description                                    | 
+| ---------------- | ---------- | ----------------------- | ---------------------------------------------- |
+| channel_id       | snowflake  | SEND_ALERT_MESSAGE      | channel to which user content should be logged |
+| duration_seconds | integer    | TIMEOUT                 | timeout duration in seconds *                  |
+
+\* Maximum of 2419200 seconds (4 weeks)
+
+
+>info
+>Only trigger type 1 is permitted to use `duration_seconds` in metadata
```

Comment:
instead of saying this for the specific metadata field, IMO it would make more sense to put something like this in action types:

> Only `KEYWORD` rules can have a `TIMEOUT` action.